netgen/ibexa-fieldtype-enhanced-link
# NGSTACK-606 add field definition section toggling

Changes
-------
- added twig content for prettier option sectioning (headings, hrs, wrappers)
- added change event handler for allowed link type choice buttons - calling it with the currently selected option on initial load/setup
    - added relevant classes to section wrappers for query selection that handle toggling
    - added admin ui css file that adds hidden class
- made it impossible for user to remove all checkbox options on "allowed link targets"
    - added relevant classes for proper container selection

Testing
---------
- `git pull` in this repo
- `yarn ibexa` in media-site project that uses this repo
- wait like 47 seconds
- edit a content type with an enhanced link field - make it two enhanced link fields
- test changing the allowed link type option (all, internal, external) - the correct sections should be visible
    - with both open, do the same - only the sections for the changed field should toggle
- test changing allowed targets change - should be unable to remove the last checkbox
